frontend/my-app/src/AdminDesktop.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {useNavigate} from "react-router-dom";
+
+export const AdminDesktop = () => {
+
+    const navigate = useNavigate()
+    const addNewProductFunction = () => {
+        navigate('/AddNewProduct')
+    }
+
+    const checkProductsFunction = () => {
+        navigate('/checkOrders')
+    }
+
+    const returnToMainPage = () =>{
+        navigate('/')
+    }
+
+
+    const checkOrdersFunction = () => {
+        navigate('/orders')
+    }
+
+    return (
+        <>
+            <hr/>

```

Comment:
take  care of formatting. Maybe "ul" would be better in this case, or "button". But not "span"